vpr/muxes/logic/mux2/simtest/Makefile

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
Is this generated by cocotb?

Reply:
I used the template and just changed a few variables. We can certainly simplify it.

Reply:
Looks like this is simplified now.